app/presenters/result_set_presenter.rb

```diff
@@ -19,7 +20,7 @@ def initialize(finder, filter_params, view_context)
 
   def to_hash
     {
-      total: total > 1000 ? "1,000+" : total,
+      total: number_with_delimiter(total, delimiter: ','),
```

Comment:
The default is `,` so not sure we need to explicitly pass in delimiter here. https://apidock.com/rails/ActionView/Helpers/NumberHelper/number_with_delimiter

Reply:
If I change it to just `total: total` it doesn't have a comma.

Reply:
@andysellick Sorry, that wasn't clear. I meant the default for `number_with_delimiter` is a comma, so not sure `delimiter` needs to be explicitly defined.

` total: number_with_delimiter(total)`

Reply:
Ah yes, right you are. Have changed, thanks.